release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1216,6 +1218,38 @@ With this release, session affinity without a timeout is possible by setting the
+
+* Previously, with oc-mirror plugin v2 (Technology Preview), mirroring errors were only displayed in the console output, making it difficult for users to analyze and troubleshoot other issues. For example, an unstable network might require a rerun, while a manifest unknown error might need further analysis to skip an image or Operator. With this update, a file is generated that contains all errors in the workspace `working-dir/logs` folder. And all the errors that occur during the mirroring process are now logged in `mirroring_errors_YYYYMMdd.txt`. (link:https://issues.redhat.com/browse/OCPBUGS-33098[*OCPBUGS-33098*])
+
+* Previously, the Cloud Credential Operator utility (`ccoctl`) could not run on a {op-system-base} 9 host with FIPS enabled. With this release, a user can run a version of the `ccoctl` utility that is compatible with the {op-system-base} version of their host, including {op-system-base} 9. (link:https://issues.redhat.com/browse/OCPBUGS-32080[*OCPBUGS-32080*])
+
+* Previously, when mirroring operator catalogs, `oc-mirror` would rebuild the catalogs and regenerate their internal cache based on `imagesetconfig` catalog filtering specifications. This process required the `opm` binary from within the catalogs. Starting with version 4.15, operator catalogs include the `opm` {op-system-base} 9 binary, which caused the mirroring process to fail when executed on {op-system-base} 8 systems. With this release, `oc-mirror` no longer rebuilds catalogs by default; instead, it simply mirrors them to their destination registries.
++
+To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However, note that no changes were made to the current implementation, so using this flag might result in the cache not being generated or the catalog not being deployed to the cluster. If you use this command, you can export `OPM_BINARY` to specify a custom `opm` binary that corresponds to the catalog versions and platform found in {product-title}. Mirroring of catalog images is now done without signature verification. Use `--enable-operator-secure-policy` to enable signature verification during mirroring.
+(link:https://issues.redhat.com/browse/OCPBUGS-31536[*OCPBUGS-31536*])
+
+* Previously, some credentials requests were not extracted properly when running the `oc adm release extract --credentials-requests` command with an `install-config.yaml` file that included the the `CloudCredential` cluster capability. With this release, the `CloudCredential` capability is correctly included in the OpenShift CLI (`oc`) so that this command extracts credentials requests properly. (link:https://issues.redhat.com/browse/OCPBUGS-24834[*OCPBUGS-24834*])
```

Comment:
```suggestion
* Previously, some credentials requests were not extracted properly when running the `oc adm release extract --credentials-requests` command with an `install-config.yaml` file that included the `CloudCredential` cluster capability. With this release, the `CloudCredential` capability is correctly included in the OpenShift CLI (`oc`) so that this command extracts credentials requests properly. (link:https://issues.redhat.com/browse/OCPBUGS-24834[*OCPBUGS-24834*])
```